docs/scos/dev/upgradability-services/upgradability-reference.md

```diff
@@ -5,15 +5,316 @@ last_updated: Nov 25, 2021
 template: concept-topic-template
 ---
 
-## Method is overridden with a Factory, Dependency Provider, Repository, or Entity Manager extended
+## Transfer, Transfer property, DB table, DB column, Method, or Constant: Name is not unique
+
+* Check that Transfer is unique and have the prefix
+* Check that Transfer extend Core Transfer, and each property have the prefix
```

Comment:
Did you mean "Check that all the properties of an extended core class have the prefix"?